packages/cwp-template-admin/index.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+
+const crypto = require("crypto");
+const fs = require("fs-extra");
+const path = require("path");
+const loadJsonFile = require("load-json-file");
+const writeJsonFile = require("write-json-file");
+const { v4: uuidv4 } = require("uuid");
+
+const s3BucketName = (projectId, appName, env) => {
+    return `${projectId}-${appName.toLowerCase().replace(/_/g, "-")}-${env}`;
+};
+
+String.prototype.toCamelCase = function() {
+    return this.replace(/^([A-Z])|[\s-_](\w)/g, function(match, p1, p2) {
+        if (p2) {
+            return p2.toUpperCase();
+        }
+        return p1.toLowerCase();
+    });
+};
+
+module.exports = async ({ appName, root }) => {
+    const appNam = appName.toCamelCase();
```

Comment:
Let's solve this in a better way. Let's move the handling of this to `create-webiny-project`, so we don't duplicate this logic in every template. Then, let's not do camelcase, but instead let's not disallow special characters and "." in the project name. Only allow `[a-zA-Z0-9-_]`. 

Handling it in one place will help us to modify this later if necessary, without updating the code in 3+ packages. Then you can safely remove this `toCamelCase` entirely from these template packages, because `appName` will already be properly formatted when it enters this function.

Reply:
problem with [a-zA-Z0-9-_] would be, s3 does not accept _ as a character in name. So we will have [a-zA-Z0-9] as name rest all is removed? or should i throw error not a valid name?

Reply:
We handle that separately so S3 buckets are safe:

```
const s3BucketName = (projectId, appName, env) => {
    return `${projectId}-${appName.toLowerCase().replace(/_/g, "-")}-${env}`;
};
```